docs/content/graphql-api-authorization.md

```diff
@@ -519,5 +519,5 @@ You learned a number of things about API authorization with OPA:
 * You can use a remote bundle server for distributing policy and data.
 
 The code for this tutorial can be found in the
-[open-policy-agent/contrib](https://github.com/open-policy-agent/contrib)
+[StyraInc/graphql-apollo-example][graphql-example-repo]
```

Comment:
👍 Let's add the repo to integrations.yml, too.

Reply:
It's not a proper integration, just a tutorial. Does that still work for `integrations.yml`?

Reply:
I thought about adding the repo [here](https://github.com/open-policy-agent/opa/blob/main/docs/website/data/integrations.yaml#L22)